gems/aws-crt/spec/auth/signer_spec.rb

```diff
@@ -29,7 +29,7 @@ module Auth # :nodoc:
 
           let(:http_request) do
             method = 'get'
-            path = 'test_uri'
+            path = 'test_uri/'
```

Comment:
We recently stopped treating empty path as` \` because of that the signature changed. This is the correct behavior because PHP faced a signature mismatch error because we treated empty path as` \`.